docs/guides/dsp-guide.md

```diff
@@ -80,6 +80,15 @@ The following table provides details for Step 2 of the workflow diagram shown in
 | 2-a | Server-side SDK (see [SDKs: Summary](../sdks/summary-sdks.md)) | Leverage the provided SDK to decrypt incoming UID2 tokens. The response contains the `UID2` and the UID2 creation time. |
 | 2-b | | DSPs are required to honor opt-out protocol for UID2s. For details on configuring user opt-outs and honoring them during bidding, see [Honor user opt-outs](#honor-user-opt-outs). |
 
+## Recommendations for Managing Latency
+
+For a low latency/high throughput setup, follow these recommendations:
+
+- Have a local instance of the UID2 client for each server. This can be in-proc or out-of-proc. In-proc is easiest.
```

Comment:
let's use the proper term in-process/out-of-process

Reply:
Done, thx.